ES/ESmy001.xml

```diff
@@ -363,129 +751,143 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                            <note/>
                            <ab type="ruling"/>
                            <ab type="pricking">Pricking and ruling are visible.</ab>
-                           <ab type="ruling" subtype="pattern"> Ruling pattern:
-                              1A-1A-1A1A/0-0/0-0/C.</ab>
-                           <ab type="pricking"> Primary pricks are partly visible (s. <locus target="#43r">43r</locus>).</ab>
-                           <ab type="pricking"> Ruling pricks are visible.</ab>
-                           <ab type="ruling"> The upper line is written below the ruling.</ab>
-                           <ab type="ruling"> The bottom line is written above the ruling.</ab>
+                           <ab type="ruling" subtype="pattern">1A-1A-1A1A/0-0/0-0/C.</ab>
+                           <ab type="pricking">Primary pricks are partly visible (see <locus target="#43r">43r</locus>).</ab>
+                           <ab type="pricking">Ruling pricks are visible.</ab>
+                           <ab type="ruling">The upper line is written below the ruling.</ab>
+                           <ab type="ruling">The bottom line is written above the ruling.</ab>
                         </layout>
```

Comment:
delete `</layout><layout>` (lines 759-760), the layout description continues

Reply:
ok